src/app/lib/utilities/getEmbedUrl/index.js

```diff
@@ -2,8 +2,10 @@ import isLive from '../isLive';
 
 const AV_ROUTE = 'ws/av-embeds';
 
-const LIVE_URL = 'https://polling.bbc.co.uk';
-const TEST_URL = 'https://polling.test.bbc.co.uk';
+const LIVE_CANONICAL_URL = 'https://bbc.co.uk';
+const LIVE_AMP_URL = 'https://polling.bbc.co.uk';
+const TEST_CANONICAL_URL = 'https://test.bbc.co.uk';
```

Comment:
Worth trying .com instead of .co.uk?

Reply:
Let's keep chat on this to the y tho thread I just wanna see that face every time I look at this PR